internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1107,6 +1108,47 @@ public void testBlobP2PTransferAmongDVC() throws Exception {
     }
   }
 
+  /**
+   * Test to verify the snapshot generation for plain table.
+   */
+  @Test(timeOut = 2 * TEST_TIMEOUT)
+  public void testDVCSnapshotGenerationForPT() throws Exception {
```

Comment:
we can use `DataProviderUtils#trueAndFalseProvider` to control whether this client uses `MEMORY_BACKED_BY_DISK` or `DISK`